docs/docs/webchat/webchat-window.md

```diff
@@ -0,0 +1,41 @@
+---
+id: webchat-window
+title: Conversation Window Format
```

Comment:
should this be a separated section? or can be included inside the customization of webchat messages? IMHO, I think it's a little bit out of context.

Reply:
Section moved